core/src/main/java/org/jvalue/ods/coreservice/rest/v1/PipelinesEndpoint.java

```diff
@@ -26,8 +26,12 @@ public PipelinesEndpoint(PipelineManager pipelineManager) {
     }
 
     @GetMapping
-    public Iterable<PipelineConfig> getPipelines() {
-        return pipelineManager.getAllPipelines();
+    public Iterable<PipelineConfig> getPipelines(@RequestParam(name = "datasourceId", required = false) Long datasourceId) {
```

Comment:
Maybe adding a comment to explain the different behavior for present/absent datasourceId parameter would make sense